source/webapp/savedRecipes/script.js

```diff
@@ -0,0 +1,64 @@
+window.addEventListener('DOMContentLoaded', init);
+window.onload = function(){
+    this.loadHome();
+}
+
+function init(){
+    let recipes = getRecipesFromStorage();
+    addRecipesToDocument(recipes);
+    for (var i = 0; i < recipes.length; i++) {
+        let ButtonEl = document.querySelectorAll("button")[i];
+        ButtonEl.addEventListener('click', () => {
+            window.location = "../CustomizeRecipe/customize.html";
+        })
+    }
+}
+
+function getRecipesFromStorage() {
+    return JSON.parse(window.localStorage.getItem('savedRecipes'));
+}
+
+function addRecipesToDocument(recipes) {
+    var tbl = document.querySelector("table");
+    for (var i = 0; i < recipes.length; i++) {
+      tbl.insertRow(-1).innerHTML = `<td><div>${recipes[i].recipeName}</div></td>
```

Comment:
this will work but is not ideal. You should not write html in js file.
use functions like table.insertRow or row.insertCell instead of passing in direct HTML